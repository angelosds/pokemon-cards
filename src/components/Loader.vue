<template>
  <div class="loader"></div>
</template>

<style lang="scss" scoped>
$border-size: 12px;
$loader-size: 120px;

.loader {
  animation: 1s spin ease-out infinite;
  border-bottom: $border-size solid $primary-color;
  border-left: $border-size solid $primary-color;
  border-radius: $loader-size;
  border-right: $border-size solid $primary-color;
  border-top: $border-size solid $secondary-color;
  height: $loader-size;
  width: $loader-size;
}
</style>
