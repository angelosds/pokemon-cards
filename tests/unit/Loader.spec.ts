import { mount } from "@vue/test-utils";

import Loader from "@/components/Loader.vue";

describe("Loader.vue", () => {
  const build = () => {
    const wrapper = mount(Loader);

    return { wrapper };
  };

  it("renders the component", () => {
    const { wrapper } = build();

    expect(wrapper.html()).toMatchSnapshot();
  });
});
