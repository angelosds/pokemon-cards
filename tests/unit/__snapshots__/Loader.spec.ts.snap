// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader.vue renders the component 1`] = `
<div class="loader">
  <div class="loader__circle"></div>
</div>
`;
